### PR TITLE
Expose barrierDismissible for confirmationDialog

### PR DESCRIPTION
**Pull request by sebastianbuechler**
Added the parameter barrierDismissible  for exposure in the confirmationDialog and adjusted the example to use the new functionality directly.

### CODE REVIEW

File: packages/stacked_services/example/lib/ui/views/home_screen.dart [hunk 1]
```diff
@@ -92,6 +92,7 @@ class _HomeScreenState extends State<HomeScreen> {
                 await _dialogService.showConfirmationDialog(
                   title: 'Test Confirmation Dialog Title',
                   description: 'Test Confirmation Dialog Description',
+                  barrierDismissible = true,
```
Comment (FilledStacks) [inline packages/stacked_services/example/lib/ui/views/home_screen.dart]: This doesn't compile

**Review by FilledStacks — CHANGES_REQUESTED** [review]
example code doesn't compile

File: packages/stacked_services/example/lib/ui/views/home_screen.dart (hunk 1 above)
Comment (sebastianbuechler) [reply packages/stacked_services/example/lib/ui/views/home_screen.dart]: Mixed up the colon notation with equal signs in two places.

**Comment by FilledStacks** [comment]
Thanks. Will push in new update.

**Comment by FilledStacks** [comment]
it's in 0.4.11

**Comment by sebastianbuechler** [comment]
Thank you

**Pull request merged**